src/containerapp/azext_containerapp/tests/latest/recordings/test_containerapp_env_dapr_components.yaml

```diff
@@ -859,8 +865,8 @@ interactions:
 - request:
     body: '{"location": "eastus", "tags": null, "properties": {"daprAIInstrumentationKey":
       null, "vnetConfiguration": null, "appLogsConfiguration": {"destination": "log-analytics",
-      "logAnalyticsConfiguration": {"customerId": "0849d8b8-7ccd-4e08-87a3-27e4d75603b4",
-      "sharedKey": "PH5FrLOtTzOnwDq1M/AWYvJm2y2sZ2H2++/ha+WpvC/6RdReJkeutxJW0DytieAF27O4/O7NTJudZu2b5xlXFQ=="}},
+      "logAnalyticsConfiguration": {"customerId": "d0118722-4b37-4428-80e4-b6252f3520fc",
+      "sharedKey": "Yozh2rH7tuhMwRklVyhB/kyuIqgWylr76Fab9s2o11ZfFrdnJDuYWDQtZ1dCoqi8WlYhbec2S62TZGmeVmth8Q=="}},
       "customDomainConfiguration": null, "workloadProfiles": null, "InfrastructureResourceGroup":
       null, "zoneRedundant": false}}'
```

Comment:
In extension version 0.3.42, we updated the cluster of environment from V1 to V2.
I notice in this recording file, the environment is still V1 cluster.
Please make sure rerun the test Pass lively with V2 cluster.